Gordon360/ApiControllers/JobsController.cs

```diff
@@ -323,5 +323,198 @@ public IHttpActionResult DeleteClockIn()
             return Ok(result);
         }
 
+        //staff routes
+
+
+        /// <summary>
+        /// Get a user's active jobs
+        /// </summary>
+        /// <param name="details"></param>
+        /// <returns>The Staff's active jobs</returns>
+        [HttpPost]
```

Comment:
I'm just following what the student time sheets did because were going to use their front end structures. I think you might be right but I don't think it would be good to change right now, due to time and lack of understanding of how their whole system works.